### ISSUE TITLE
Bug: Remove Form Saving section in header in Admin Application view screens

### ISSUE THREAD

**Issue by jmmcnj** [18F/dol-whd-14c#125]

**Comment by rrefoy** [comment]
@jmmcnj can you flesh this out a bit more so we understand?

**Comment by jmmcnj** [comment]
yup, so we noticed that the saved notice is coming up on the admin views/pages, they shouldnt because it is not saving in admin mode, so we were just going to remove it altogether.  It seems to be popping up after you access the Wage Data page, @klinden or @jefferey could explain better what needs to be done.  I believe they removed it already but it is sneaking in there through a save function on one of the views (i.e. - Wage Data)
